packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -473,6 +473,9 @@ export interface IndexerProductPayment {
   // Represents the annually interest rate for spots and annually funding rate for perps.
   annualPaymentRate: BigDecimal;
   oraclePrice: BigDecimal;
+  isolated: boolean;
+  // Only defined when primaryQuoteProduct (productId=0) and isolated=true otherwise null.
```

Comment:
Can we reuse the same comment as we have currently in `serverModelTypes`?

`// The product ID associated with the isolated perp market. This is only used when product_id === QUOTE_PRODUCT_ID and isolated === true`